src/examples/dac/dac.go

```diff
@@ -0,0 +1,40 @@
+// Simplistic example using the DAC on the Circuit Playground Express.
+//
+// To actually use the DAC for producing complex waveforms or samples requires a DMA
+// timer-based playback mechanism which is beyond the scope of this example.
+package main
+
+import (
+	"machine"
+	"time"
+)
+
+func main() {
+	enable := machine.PA30
+	enable.Configure(machine.PinConfig{Mode: machine.PinOutput})
+	enable.Set(true)
+
+	speaker := machine.A0
+	speaker.Configure(machine.PinConfig{Mode: machine.PinOutput})
+
+	machine.DAC0.Configure(machine.DACConfig{})
+
+	data := []uint16{4096, 2058, 1024, 512, 256, 128}
+
+	for {
+		for _, val := range data {
+			play(val)
+			time.Sleep(500 * time.Millisecond)
+		}
+	}
+}
+
+func play(val uint16) {
```

Comment:
Actually, this example does play an audio tone at 4 different amplitudes using the Circuit Playground Express built-in speaker.

Reply:
Huh that's strange. What kind of speaker is that? Wouldn't a speaker normally need a sine wave (or something) to play a tone instead of a particular voltage?

Reply:
The loop turns the power on and off acting like a very simple wave ala PWM. Of course just setting a single voltage would not produce any sound! :)